packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -3813,7 +3813,7 @@ describe('ReactDOMFizzServer', () => {
         'Logged recoverable error: There was an error while hydrating this Suspense boundary. Switched to client rendering.',
       ]);
     }).toErrorDev(
-      'Warning: Prop `name` did not match. Server: "initial" Client: "replaced"',
+      'Warning: Text content did not match. Server: "initial" Client: "replaced',
```

Comment:
We now check the `children` before the rest of the attributes, and then error. We could maybe check all the attributes before throwing the error so that you get more logs for context.